### DISCUSSION TITLE
Feature Request: Filtered Queries In Event Triggers

### DISCUSSION THREAD

**Discussion by eaf-drye** [hasura/graphql-engine#8639] [General]
Hello all,

This is a request to add query filters to the Pending Events/Processed Events/Invocation Logs on the Event Triggers dashboard similar to the filtered queries we can currently execute against our database schemas/tables.  i.e. the ability to query against triggers that are in an error status or events that ran on a particular day, etc.

Thank you!

**Comment by meetzaveri** [comment]
Hello @drye-farms 

Thanks for submitting a feature idea. We would recommend you to create a github issue on this with subject of feature request and follow it. That would be a good starting point and our team would be aware of it for tracking it in the future. You can easily do this via clicking [here](https://github.com/hasura/graphql-engine/issues/new?created_from_discussion_number=8639) or clicking "Create issue from discussion" which is present in the bottom of the right sidebar of this discussion page.

Cheers!
